packages/lib/src/dependency/arc/migration.ts

```diff
@@ -73,8 +73,8 @@ export const migrateDAO = async (
       return { receipt, result };
     };
 
-    const arcVersion = "0.1.2-rc.0";
-    const getArcVersionNumber = (ver: string) => Number(ver.slice(-2));
+    const arcVersion = "0.1.2-rc.2";
+    const getArcVersionNumber = (ver: string) => Number(ver.slice(-1));
```

Comment:
We are adding this change because with `slice(-2)` we are getting the following error: `invalid number value (arg="", coderType="uint64", value=0.2)`